sql/core.go

```diff
@@ -836,8 +836,6 @@ func (s *ImmutableStatusVarValue) Copy() StatusVarValue {
 // name is case-insensitive. Errors are ignored.
 // This runs in a goroutine to avoid blocking the caller, but we do not wait for it to complete.
```

Comment:
Since this no longer runs in a go routine, remove this comment

Reply:
👍